lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,417 @@
+                      "code": "3693"
+                    },
+                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                  }
+                ],
+                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-ICD",
+                "code": "R51"
+              }
+            ]
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the ICD proprietary Bill Codes Types systems.</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.reason
+
+  - name: Replacing Extension
+    required: 'Yes'
+    type: Extension
+    description: The resource that this ChargeItem is replacing.
+    action: modify
+    example: |
+      {
+        "extension": [
+          {
+            "valueId": "1",
+            "url": "item-version"
+          }
+        ],
+        "valueReference": {
+          "reference": "ChargeItem/317370800"
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/replacing"
+      }
+    note: >
+      <ul>
+        <li>Can only provide one replacing extension</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: performing-location
+    required: 'No'
+    type: Extension
+    description: A location where the charge was performed.
+    action: modify
+    example: |
+      {
+        "valueReference": {
+          "reference": "Location/21304903",
+          "display": "NM"
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/performing-location"
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: description
+    required: 'No'
+    type: Extension
+    description: A description providing additional details of the resource.
+    action: modify
+    example: |
+      {
+        "valueString": "Acetaminophen Level",
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/description"
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: custom-attribute
+    required: 'No'
+    type: Extension
+    description: A client defined custom attribute for the charge.
+    action: modify
+    example: |
+      {
+        "extension": [
+          {
+            "id": "2572130093",
+            "valueString": "Special Authority",
+            "url": "custom-attribute-name"
+          },
+          {
+            "valueInteger": 1,
+            "url": "custom-attribute-value"
+          }
+        ],
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
+      }
+    note: >
+      <ul>
+        <li>Custom-attribute-name and Custom-attribute-value are both required</li>
+        <li>Custom-attribute-value can be of type integer, string, or decimal</li>
```

Comment:
nit: lower case 'c' on the extension names.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/4595890bdc6f918a94d93cd54ca164dab864e8e0#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R278-R279